tests/app/celery/test_letters_pdf_tasks.py

```diff
@@ -44,6 +44,7 @@
 from tests.conftest import set_config_values
 
 
+@pytest.mark.skip(reason="Letter tests")
```

Comment:
I like that you can add a reason here, very cool! Is there a way to skip all the tests in this file with one statement, since they are all letter tests?

Reply:
Good question, I'm not too sure.  I skipped this one specifically because it is now failing.